gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -525,44 +495,30 @@ properties:
       - Not Reported
       - Not Allowed To Collect
 
-  hiv_status:
+  hiv_positive:
     term:
-      $ref: "_terms.yaml#/hiv_status"
+      $ref: "_terms.yaml#/hiv_positive"
     enum:
-      - Equivocal
-      - Indeterminate
-      - Not Done
-      - Not Available
-      - Negative
-      - Positive
+      - "Yes"
```

Comment:
Is there a reason this is an enum, not a `boolean`? Do we expect more non-binary fields in the future?